bundle/bind.go

```diff
@@ -37,24 +44,49 @@ func (e *executor) Bind(
 
 	go func() {
 		e.actionStarted()
-		executionContext, err := e.executeApb(
-			"bind", instance.Spec, instance.Context, parameters)
+		// Create namespace name that will be used to generate a name.
+		ns := fmt.Sprintf(fmt.Sprintf("%s-%.4s-", instance.Spec.FQName, bindAction))
```

Comment:
I'm sure there's a reason for doing the double `fmt.Sprintf` but would like to hear what it is :)